fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -463,7 +464,7 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         // We need the push feature setup here to deliver messages in the case where the service
         // starts up the app first.
         components.push.feature?.let {
-            Logger.info("AutoPushFeature is configured, initializing it...")
+            logger.info("AutoPushFeature is configured, initializing it...")
```

Comment:
Cleaning up some of these individual `Logger` to prefer our tagged Logger defined in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt#137